_posts/2019-04-25_node_version_menager.md

```diff
@@ -0,0 +1,77 @@
+layout:    post
+title:     "Korzyści wynikające użycia Node Version Manager"
+date:      2019-04-25 10:11:00 +0100
```

Comment:
wpiszmy piątkową datę, omówimy to na chapterze i jak będzie git to publikujemy

Reply:
zrobione